packages/app/src/components/Settings/Settings.js

```diff
@@ -194,59 +214,114 @@ const Settings = (props) => {
           <div className="settings__input-block">
             <p className="settings__input-label">Start Date:</p>
             <span className="settings__input">
-              <input
-                value={newGrantCycle.openedOn}
-                name="openedOn"
-                onChange={handleChange}
-                type="date"
-              />
+              <div
+                style={
+                  showEditModal
+                    ? {
+                        display: 'none',
+                      }
+                    : {
+                        position: 'relative',
+                      }
+                }
+              >
+                <DatePicker
+                  name="openedOn"
+                  value={newGrantCycle.openedOn}
+                  selected={newGrantCycle.openedOn}
+                  onChange={handleStartDateCreation}
+                  placeholderText="mm/dd/yyyy"
+                />
+              </div>
             </span>
           </div>
           <div className="settings__input-block">
             <p className="settings__input-label">End Date:</p>
             <span className="settings__input">
-              <input
-                value={newGrantCycle.closedOn}
-                name="closedOn"
-                onChange={handleChange}
-                type="date"
-              />
+              <div
+                style={
+                  showEditModal
+                    ? {
+                        display: 'none',
+                      }
+                    : {
+                        position: 'relative',
+                      }
+                }
+              >
+                <DatePicker
+                  name="closedOn"
+                  value={newGrantCycle.closedOn}
+                  selected={newGrantCycle.closedOn}
+                  onChange={handleEndDateCreation}
+                  placeholderText="mm/dd/yyyy"
+                />
+              </div>
             </span>
           </div>
           <div className="settings__input-block">
             <p className="settings__input-label">Cycle Name:</p>
             <span className="settings__input">
-              <input
-                value={newGrantCycle.name}
-                name="name"
-                onChange={handleChange}
-                type="text"
-              />
+              <div
+                style={
+                  showEditModal
+                    ? {
+                        display: 'none',
+                      }
+                    : {
+                        position: 'relative',
+                      }
+                }
```

Comment:
Same feedback here as in the other comment(s) regarding not using inline styles.